ngViewer/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html ng-app="ngViewer">
+<head>
+
+    <meta charset="utf-8">
+
+    <!--CSS-->
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css"/>
+    <link rel="stylesheet" type="text/css" href="css/bootstrap-theme.min.css"/>
+    <link rel="stylesheet" type="text/css" href="font-awesome/css/font-awesome.min.css"/>
```

Comment:
I didn't notice that the non-minified version was there. Anyways it's gone now...